docs/examples/fenum-extension/FenumDemo.java

```diff
@@ -43,6 +52,11 @@ void foo() {
         myFenumArg(TestStatic.CCONST1); // ok
     }
 
+    @DefaultQualifier(MyFenum.class)
+    void bar() {
+        int int0 = TestStatic.CCONST1; // ok, @MyFenum applies by default in this method
```

Comment:
```suggestion
        int int0 = TestStatic.CCONST1; // ok, @MyFenum applies by default in this method
        @NonNegative int int1 = TestStatic.BCONST1; // ok, @MyFenum applies by default in this method
```

Reply:
Does the default get applied if there is a `@NonNegative`? If not, can you add it as a TODO and open a follow-up issue?